domino/domino.py

```diff
@@ -925,6 +929,74 @@ def model_version_publish(
         response = self.request_manager.post(url, json=request)
         return response.json()
 
+    # Dataset Functions
+    def datasets_list(self, project_id=None):
+        self.requires_at_least("3.6.0")
```

Comment:
"3.6.0" seems to be used in several places, perhaps it could be a constant?

Reply:
As 3.6.x is the current minimum version available, that will be set as min required, until 3.6 is fully deprecated.